pkg/webui/console/components/pubsub-form/mapping.js

```diff
@@ -63,14 +63,16 @@ export const mapPubsubToFormValues = function(pubsub) {
     _provider: isMqtt ? 'mqtt' : 'nats',
     nats: isNats ? mapNatsServerUrlToFormValue(pubsub.nats.server_url) : natsBlankValues,
     mqtt: isMqtt ? pubsub.mqtt : mqttBlankValues,
-    uplink_message: mapPubsubMessageTypeToFormValue(pubsub.uplink_message),
-    join_accept: mapPubsubMessageTypeToFormValue(pubsub.join_accept),
     downlink_ack: mapPubsubMessageTypeToFormValue(pubsub.downlink_ack),
-    downlink_nack: mapPubsubMessageTypeToFormValue(pubsub.downlink_nack),
-    downlink_sent: mapPubsubMessageTypeToFormValue(pubsub.downlink_sent),
     downlink_failed: mapPubsubMessageTypeToFormValue(pubsub.downlink_failed),
+    downlink_nack: mapPubsubMessageTypeToFormValue(pubsub.downlink_nack),
```

Comment:
The larger diff here is because I sorted the topics. If you feel the order shouldn't be changed I can only add the new fields.

Reply:
No worries.